api/src/main/java/org/pmiops/workbench/utils/CohortMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.utils;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import org.pmiops.workbench.db.model.DbCohort;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.Cohort;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class},
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface CohortMapper {
+  DbCohort clientToDbModel(Cohort source);
+
+  Cohort dbModelToClient(DbCohort destination);
+
+  default DbUser DbUserToCreatorEmail(String creator) {
+    final DbUser result = new DbUser();
+    result.setEmail(creator);
+    return result;
```

Comment:
Something seems off about creating incorrect models like this.  I can very easily see this leading to a bug where a someone calls any non email field like `dbCohort.creator.id` and result in an NPE.

We can use DAOs to do this mapping.

Reply:
If we can use the DAO, then why do we need the mapper?

We don't have very much checking at all for required parameters on any of our models. I have some thoughts about that, but are you suggesting the mapper class should make a Dao call?

Reply:
Yeah. MapStruct is integrated into Spring's DI so you can pull in the UserDao and I think it will know how to convert String -> User. If not automatically, it would be a small function or annotation to add the mapping.

Reply:
Yeah, it just worked. And I noticed it's smart enough to tell me when there's more than one function that would do the job, so it's a win-win.